### PR TITLE
Coordination meeting minutes, 2020-03-26

### PR DESCRIPTION
**Pull request by graeme-a-stewart**
Usual minutes from today's meeting

### CODE REVIEW

File: organization/_posts/2020/2020-03-26-coordination.md [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+Apologies: Ben Morgan
+
+## News, general matters
+
+### Discussion with CMS
+
+We were invited by Danilo and Markus to give a talk to CMS in a [session on external projects](https://indico.cern.ch/event/902343/) (along with IRIS-HEP and HEP-CCE)
+- Outline of the HSF, motivation and history
+- Current structure and activities
+- Propsects for future collaboration and impact
```
Comment (amete) [inline organization/_posts/2020/2020-03-26-coordination.md]: Propsects => Prospects

File: organization/_posts/2020/2020-03-26-coordination.md [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+
+The "[Software Institute for Data Intensive Sciences (SIDIS)](https://sidis.web.cern.ch/)" is an initiative to establish connections and promote common R&D between software engineers in natural sciences and computer science departments at universities, currently focusing on European partners. SIDIS currently is in a conceptualisation phase. 
+
+The activities of SIDIS are regularly discussed on the **third Wednesday of the month, 10.00 - 11.00 CE(S)T**, in the [SIDIS Coordination meeting](https://indico.cern.ch/category/11734/). The meeting usually contains topics to discuss and is public. If you are interested in a specific topic or SIDIS in general please join. 
+
+Current activities of SIDIS are:
+- the "Alpaka training and hackathon (see below)" in collaboration with HSF, Openlab and [CASUS](https://www.hzdr.de/db/Cms?pNid=1227&pLang=en)
+- a "Topical Workshop on accelerator programming" tentatively to be held in early December with the goals to
+    - Draw a landscape of current national initiatives similar to SIDIS and connecting those
+    - Exercise the core idea of SIDIS to connect computer scienctists and natural scientists, aiming at interactivity as much as possible
```
Comment (amete) [inline organization/_posts/2020/2020-03-26-coordination.md]: scienctists => scientists

File: organization/_posts/2020/2020-03-26-coordination.md [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+### HSF Feedback Process
+
+**Discussion [with CMS](https://indico.cern.ch/event/902139/)** happened on Tuesday 24, points that came up were:
+
+- Is there an overall plan and priority?
+    - No and we did not expect to have one at this stage
+    - Prioritisation can happen on a profiled basis (invest more in the more expensive areas); *however every experiment is at a different working point, so no one-size-fits-all recommendation*
+- HSF should comment on the performance of generators, not on the decisions of the experiments
+    - Indeed, but comparison of ATLAS and CMS led to significant insights in this area; it is not a competitive or pejorative comparison - this was the spirit of the study, which was led by ATLAS and CMS generator experts
+- WLCG does not foresee 'analysis facilties' being distinguished at the resource accounting level (they stick to the Tier 0/1/2 model)
```
Comment (amete) [inline organization/_posts/2020/2020-03-26-coordination.md]: facilties => facilities

File: organization/_posts/2020/2020-03-26-coordination.md [hunk 4]
```diff
@@ -0,0 +1,209 @@
...
+- Setup a semi-permanent chat channel (not just VC chat), like Slack or Mattermost.
+- [discussion needed] material can be indexed on Zenodo, including recordings
+    - Practically, use Indico to get recording permission
+    - Not planning to pre-record talks, but to have material still presented during the event
+
+*This would be a learning experience for the community, that could lead the way towards less travel and f2f meetings as well as keeping activities running in times of COVID-19.*
+
+- **Is this worthwhile doing?** Please *do* give feedback on whether you believe this event is good to organise and if you would attend it (in particular, if your feelings are *negative*)
+    - Liz - virtual meetings need to have a specific focus, but experience of virtual CMS S&C week is positive
+    - David - *Conneting the Dots* will be pre-recording material (otherwise not practical to do 20hrs of material from many timezones), so this will provide some experience of an alternative format
```
Comment (amete) [inline organization/_posts/2020/2020-03-26-coordination.md]: Conneting => Connecting

File: organization/_posts/2020/2020-03-26-coordination.md [hunk 5]
```diff
@@ -0,0 +1,209 @@
...
+- In touch with SciPy 2020 conference chairs for what happens there.
+
+## AOB
+
+We changed to Zoom this week, a regular meeting has been booked until the end of the year
+- Please give [feedback on your Zoom experience](https://espace.cern.ch/AVC-workspace/videoconference/Lists/Zoom%20Pilot%20Feedback/overview.aspx) to CERN IT
+
+### Next Meeting
+
+- Next regular meeeting slot is 9 April
```
Comment (amete) [inline organization/_posts/2020/2020-03-26-coordination.md]: meeeting => meeting

**Comment by graeme-a-stewart** [comment]
Thanks a lot @amete - always very useful to have someone else check!

**Pull request merged**